### ISSUE TITLE
WasmEdge 0.10.0 not working in windows

### ISSUE THREAD

**Issue by Puellaquae** [WasmEdge/WasmEdge#1559]
## Description

wasmedge.exe output nothing and exit with code -1073740791. Version 0.10.0, 0.10.0-rc.1, 0.10.0-alpha-2 in releases all not work.

## Environment

Windows 10 Pro 21H2 19044.1706

## Steps to Reproduce

just exec wasmedge.exe

**Comment by juntao** [comment]
Can you attach the wasm file you are trying to run? Thanks.

**Comment by calvin2021y** [comment]
or you can package msvcp140.dll  into release package.

**Comment by hydai** [comment]
Thanks, @Pospelove provided an awesome PR #2455 to fix this. However, the LLVM we used(llvm-windows) currently doesn't offer the MT mode. If we applied #2455 would make the Windows CI fail. I kept the variable `CMAKE_MSVC_RUNTIME_LIBRARY`, so users can set it to `"MultiThreaded$<$<CONFIG:Debug>:Debug>"` with disabling the AOT feature while building WasmEdge to avoid `msvcp140.dll` issue.

Currently, we are looking for how to make an `MT` mode LLVM. Before this is fixed, I will keep this issue open.
